canonical-websites/www.ubuntu.com
# Build community section

## Done

- Built new community pages and moved some from /about:
  - /community
  - /community/canonical
  - /community/code-of-conduct
  - /community/debian
  - /community/diversity
  - /community/governance
  - /community/mission

- Updated /about with new [copy](https://docs.google.com/document/d/1NvYOhOJzdY5ypifTPs73vH5Nb7rFABRWMzjXyDUQt_U/edit#heading=h.legxy5p18gzw)
- Moved /about/licensing to /licensing

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check there is a link to the new community pages in the footer
- Check that the secondary nav breadcrumbs work as intended
- Check all new community pages are present
- Check that the licensing page now lives at /licensing

## Issues

Fixes #2724, fixes #2725
